test/integration/sonobuoy_integration_test.go

```diff
@@ -32,6 +34,40 @@ func findSonobuoyCLI() (string, error) {
 	return sonobuoyPath, nil
 }
 
+// TestRunAndDelete runs a simple plugin to check that it runs successfully and can be deleted
+func TestRunAndDelete(t *testing.T) {
+	var stdout, stderr bytes.Buffer
+
+	ns := randomNamespace()
+	ctx, cancel := context.WithTimeout(context.Background(), 45*time.Second)
+	defer cancel()
+
+	runCommand := exec.CommandContext(ctx, sonobuoy, "run", "--image-pull-policy", "IfNotPresent", "--wait", "-p", "testdata/basic_plugin.yaml", "-n", ns)
```

Comment:
FWIW if you don't require special spacing in your values it can be easier to write/read if you just do `strings.Fields("run --image-pull-policy=IfNotPresent ...")`

Reply:
I didn't know about that. Thanks! :)